k8s/charts/openebs/README.md

```diff
@@ -1,22 +1,55 @@
+------------------------------------------------------------------------------
+IMPORTANT!!
+
+DEPRECATION NOTICE:
+
+The support for this chart will be discontinued soon. Please plan to migrate
```

Comment:
It is possible we will have to continue this for a little longer than 0.5.5. Depends on getting the write access to the kubernetes repo and keeping the k8s repo upto date. Also, we need to consider when we discontinue this charts, if there are any users using this chart in repo, what will be the migration path for them to move to k8s stable charts. This will be tracked under #1573